scripts/apps/says.js

```diff
@@ -59,20 +60,15 @@ import {BYPASSNAMETYPES} from './constants.js';
 
     static findSays(tokenName, actor ={}, documentType, documentName, isActive = true){
         const sys = isActive ? this.saysActive : this.says;
-        return sys.filter(sy => 
-            sy.documentType === documentType 
-            && (
-                !sy.name 
-                || (!sy.reverse && sy.nameList.includes(((sy.isActorName && actor.name) ? actor.name : tokenName)))
-                || (sy.reverse && !sy.nameList.includes(((sy.isActorName && actor.name) ? actor.name : tokenName)))
-            )
-            && (
-                !sy.documentName 
-                || BYPASSNAMETYPES.includes(documentType)
-                || sy.documentNameList.includes(documentName) 
-            )
+        return sys.filter(sy => {
```

Comment:
I expanded this out to help me read through what was happening.  It could be condensed again if you want.